src/Raven.Server/Documents/Revisions/RevisionsStorage.cs

```diff
@@ -1453,7 +1454,7 @@ private class Parameters
             public Action<IOperationProgress> OnProgress;
         }
 
-        public async Task<IOperationResult> RevertRevisions(DateTime before, TimeSpan window, Action<IOperationProgress> onProgress, OperationCancelToken token)
+        public async Task<IOperationResult> RevertRevisions(DateTime before, TimeSpan window, Action<IOperationProgress> onProgress, OperationCancelToken token, HashSet<string> collections = null)
```

Comment:
should be used as a 3rd parameter, it dos not need to have a default value of null

Reply:
There's a lot of calls to this method, so I thought it would be better to put null in there as a default value instead of changing all of the calls.
Do you still want me to change it, or do you prefer to leave it as it is instead?

Reply:
I switch between the 'collections' and the 'token' and created an overload with no 'collections' parameter..